scripts/gha/integration_testing/xcode_tool.rb

```diff
@@ -54,6 +56,10 @@ def main
       'Path to entitlements (optional)') do |entitlement_path|
       @entitlement_path = entitlement_path
     end
+    opts.on('-i', '--XCodeCPP.include [include_path]',
+      'Path to additional header search files (optional)') do |include_path|
```

Comment:
Nit: this is confusing. Should it be like above, "Path to additional include files (optional)" ?

Reply:
Fixed.